docs/flow diagrams/create-resource.md

```diff
@@ -0,0 +1,34 @@
+```mermaid
+sequenceDiagram
+    participant Client
+    participant FhirServer
+    participant Middleware
+    participant FhirController
+    participant Mediatr
+    participant CreateResourceHandler
+    participant AuthorizationService
+    participant ResourceReferenceResolver
+    participant ResourceWrapperFactory
+    participant SearchIndexer
+    participant FhirDataStore
+    Client->>FhirServer: POST Resource
+    FhirServer->>Middleware:
+    Middleware->>FhirController:
+    FhirController->>Mediatr: Create Resource
+    Mediatr->>CreateResourceHandler: Resource
```

Comment:
nit: Can this be `CreateResourceRequest` to match with the `UpsertResourceResponse` that is returned to the `Mediatr` below ?